cdk/domino_cdk/config/template.py

```diff
@@ -52,16 +51,17 @@ def add_nodegroups(name, count, min_size, instance_types, labels, disk_size=disk
     add_nodegroups(
         "platform",
         platform_nodegroups,
-        platform_min_size,
+        1,
         [platform_instance_type],
         {"dominodatalab.com/node-pool": "platform"},
     )
     add_nodegroups(
         "compute",
         compute_nodegroups,
-        1,
+        0,
```

Comment:
I vaguely remember something like 0 is not a valid value

Reply:
For managed groups, yeah, there's still some pending cluster-autoscaler support. For unmanaged, it seems to work fine.